Makd.mak

```diff
@@ -267,6 +270,9 @@ PKGVERSION := $(shell echo $(VERSION) | \
 # http://www.gnu.org/software/make/manual/make.html#Text-Functions
 TEST_FILTER_OUT := $C/$(SRC)/%/main.d
 
+# Directories to search for files to convert using d1to2fix
+D1TO2FIX_DIRS := $C
```

Comment:
Should this be `:=` or `?=` ? Since if you are using this variable, there is no way you want to add to it at the moment (since everything is being converted by default).